### PR TITLE
redux implementation for listing

### PR DESCRIPTION
**Pull request by KaveriMoger**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,9 +9,15 @@
     "eslint-config-airbnb": "^17.1.0",
     "eslint-config-prettier": "^4.1.0",
     "eslint-plugin-jsx-a11y": "^6.2.1",
+   "font-awesome": "^4.7.0",
```
Comment (Sushanthbc) [inline package.json]: fix the indentation

File: src/App.jsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ class App extends Component {
 		this.state = {
 			name: "",
 			age: "",
-			data
+			// data
```
Comment (Sushanthbc) [inline src/App.jsx]: If this is not required lets go ahead and remove it

File: src/DataList.jsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+	};
+};
+
+
+const ConnectedList = ({articles}) => (
+  <div>
+    {articles.map(el => (
+      <div>
+        <h1 key={el.id}>
+Name:
```
Comment (Sushanthbc) [inline src/DataList.jsx:15]: I think there problem with indentation

File: src/DataList.jsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import { connect } from "react-redux";
```
Comment (Sushanthbc) [inline src/DataList.jsx:2]: what does connect do here?

File: src/App.jsx (hunk 2 above)
Comment (KaveriMoger) [reply src/App.jsx]: removed

**Review by Sushanthbc — APPROVED** [review]

File: src/DataList.jsx (hunk 4 above)
Comment (KaveriMoger) [reply src/DataList.jsx:2]: connect our application to redux store

**Pull request merged**